src/services/utils.ts

```diff
@@ -3,18 +3,17 @@
  *
  * @author Katia Aresti
  */
-import { SelectOptionObject } from '@patternfly/react-core/src/components/Select/SelectOption';
 import { KeycloakService } from './keycloakService';
 
 export enum KeyContentType {
-  OctetStream = 'application/octet-stream',
-  OctetStreamHex = 'application/octet-stream; encoding=hex',
   StringContentType = 'java.lang.String',
   DoubleContentType = 'java.lang.Double',
   IntegerContentType = 'java.lang.Integer',
   LongContentType = 'java.lang.Long',
   BooleanContentType = 'java.lang.Boolean',
-  BytesType = 'Bytes'
+  BytesType = 'Bytes',
+  OctetStream = 'application/octet-stream',
+  OctetStreamHex = 'application/octet-stream; encoding=hex'
```

Comment:
Similar as above, ```byte[] as Hex```

Reply:
Which is the  list I should add ? from the console the octet-stream makes sense ?

Reply:
and If I add a marshalled 'Person' protostream type, how should I deal it? is it possible ?

Reply:
It is not possible to add specific protobuf types to the key content type header. 

> Which is the list I should add 

The same list you have know, but removing ```Bytes``` and adding  ```byte[] as Base64``` and ```byte[] as Hex```

Reply:
> Which is the list I should add

You are sending the headers correctly, but the change I suggest is to display it differently (byte[] as Hex)  and (byte[] as Base64)

Reply:
great, done and pushed